ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/commands.proto

```diff
@@ -58,6 +58,12 @@ message Commands {
   // Required
   repeated Command commands = 8;
 
+  // The time to live (TTL), in seconds.
+  // In the time window between the first submission of a command and TTL seconds later,
+  // all commands with the same application ID and command ID will be deduplicated, i.e.,
```

Comment:
Command deduplication is only based on `party` and `commandId`. ApplicationId shouldn't play a role in that.

Reply:
Thanks, description fixed